test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -97,13 +97,49 @@ describe('expand base', function () {
     ['(2x + 3)^2', '(2x + 3) * (2x + 3)'],
     ['(x + 3 + 4)^2', '(x + 3 + 4) * (x + 3 + 4)'],
     // These should not expand
-    // Needs to have a positive integer exponent > 1
     ['x + 2', 'x + 2'],
-    ['(x + 2)^-1', '(x + 2)^-1'],
     ['(x + 1)^x', '(x + 1)^x'],
     ['(x + 1)^(2x)', '(x + 1)^(2x)'],
     ['(x + 1)^(1/2)', '(x + 1)^(1/2)'],
     ['(x + 1)^2.5', '(x + 1)^2.5'],
+    ['1 / (x + 1)^x', '1 / ((x + 1)^x)']
+  ];
+
+  tests.forEach(t => testDistribute(t[0], t[1]));
+});
+
+describe('distribute negative exponent', function () {
+  const tests = [
+    ['(x y)^-1', '1 / ((x * y)^1)'],
+    ['(x y)^-x', '1 / ((x * y)^x)'],
+    ['(x y)^(-2x^2)','1 / ((x * y)^(2x^2))'],
+    ['(x y)^(-(x + 1))', '1 / ((x * y)^((x + 1)))'],
+  ];
+
+  tests.forEach(t => testDistribute(t[0], t[1]));
+});
+
+describe('distribute exponent', function () {
+  const tests = [
+    // When terms are polynomialTerms
```

Comment:
Omg Anthony this pull is amazing ... so many docstrings and test cases ... almost like you knew me and Evy would bug you about those 😁